selfdrive/car/torque_data/params.yaml

```diff
@@ -77,7 +77,8 @@ TOYOTA HIGHLANDER 2020: [2.022340166827233, 1.6183134804881791, 0.14592306380054
 TOYOTA HIGHLANDER HYBRID 2018: [1.752033, 1.6433903296845025, 0.144600]
 TOYOTA HIGHLANDER HYBRID 2020: [1.901174, 2.104015182965606, 0.14447040132184993]
 TOYOTA MIRAI 2021: [2.506899832157829, 1.7417213930750164, 0.20182618449440565]
-TOYOTA PRIUS 2017: [1.746445, 1.5023147650693636, 0.151515]
+TOYOTA PRIUS 2017: [1.60, 1.5023147650693636, 0.151515]
+TOYOTA PRIUS 2017 x060: [1.746445, 1.5023147650693636, 0.17]
```

Comment:
this isn't used anywhere?

Reply:
https://github.com/commaai/openpilot/pull/26626